src/notification/index.js

```diff
@@ -77,6 +78,10 @@ export const getAccountFromNotificationData = (
       realm_uri,
       parsed_url: realmUrl,
       match_count: urlMatches.length,
+      known_urls: knownUrls,
+      unique_identities_count: urlMatches
+        .map(matchIndex => `${identities[matchIndex].realm.origin} ${identities[matchIndex].email}`)
```

Comment:
Now that we're using `urlMatches` here, I believe the `origin` part of this is redundant. If so, better to remove it so the reader doesn't wonder if they're missing something.